database/migration/V20160724_1239__createPasswordTokenTable.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE `tokens` (
+  `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,
+  `user_id` INT UNSIGNED NOT NULL,
+  `type` ENUM('AUTH', 'OTHER') NOT NULL,
+  `value` VARCHAR(255) NOT NULL,
+  `created` DATETIME NULL DEFAULT CURRENT_TIMESTAMP,
+  PRIMARY KEY (`id`),
+  FOREIGN KEY (`user_id`) REFERENCES `users` (`id`)
```

Comment:
So technically it's not required, but good practice: http://stackoverflow.com/questions/3650690/should-every-sql-server-foreign-key-have-a-matching-index.

I'll add an index in